hofstra/coyote
# Addressed visibility of links on browse page, refactored CSS to SASS

Addresses #3  

## What does this do
1. Refactors CSS to SASS sitewide

2. Addresses the link visibility issue by: 
• Changing the indentation of the list for better visual grouping
• Changing the type size 
• Adding hover-state to the links (Rollovers are blue)
• Injecting an arrow to the end of the links

## How to test
1. This is a code change but should otherwise be invisible. So: compare layout/type to existing site and should be exactly the same

2. Visit: http://127.0.0.1:4000/coyote/browse/ 
Roll over links
